select_builder.go

```diff
@@ -84,13 +91,45 @@ func (tx *Tx) SelectBySql(query string, value ...interface{}) SelectBuilder {
 	}
 }
 
+func (b *selectBuilder) changeTimezone(value reflect.Value) {
+	if b.timezone == nil {
```

Comment:
can we put this in the upper func, to make reflect.ValueOf calls?

Reply:
ok